recipes/tinycthreadpool/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from conan import ConanFile
+from conans import CMake, tools
+from conan.tools.files import get
+
+required_conan_version = ">=1.53.0"
+
+#
+# INFO: Please, remove all comments before pushing your PR!
+#
+
+
+class PackageConan(ConanFile):
+    name = "tinycthreadpool"
+    version = "1.0"
+    description = "Portable C thread pool"
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case not listed there, use "LicenseRef-<license-file-name>"
+    license = "https://github.com/bennyhuo/tinycthreadpool/blob/master/LICENSE"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bennyhuo/tinycthreadpool"
+    # no "conan" and project name in topics. Use topics from the upstream listed on GH
+    topics = ("C", "Thread Pool", "tinycthread")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake"
```
Consider migrating it to Conan 2.0. The `cmake` generator is outdated and will become deprecated very soon. https://docs.conan.io/en/latest/conan_v2.html

Reply:
It seems that the 'generators' is needed for conan 1.x, right?

Reply:
If I simply remove this line, it won't work. And I don't know how to make it compile. Maybe we should move on to conan 2.0 later on.

Reply:
Ah, I see. I've just changed to 'CMakeDeps'.